### DISCUSSION TITLE
2 Logos light/dark mode Premium theme

### DISCUSSION THREAD

**Discussion by treeandcoffee** [GetPublii/Publii#1919] [Ideas]
Hi
Today its possible to include a logo in a premium theme, theme lignes in my case. 
Currently a user can switch between light/dark mode, thats supported with most themes. 

If you have a white background and a logo with a with a transparent background and with black typo in the designed logotext or wordmark in light mode, then as a consequence the logo and the text will not be visible in dark mode, because an uploaded logo is today prepared only for one of both modes., mostly light mode. 

For the mode switching it is a must to upload a second logo, which supports the dark mode (inverse colors or so). And the correct code.
Thats missing currently.

**Comment by bobmitro** [comment]
Thank you for your suggestion. 
We don't plan to introduce an option for uploading separate logos for light and dark modes. Most logos are vector graphics (e.g., SVG), which can be easily styled using CSS to adapt to both modes e.g.:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" width="100" height="100">

  <style>

    /* Styling for light mode */
    @media (prefers-color-scheme: light) {
      .bg {
        fill: white; /* Background is white in light mode */
      }
      .circle {
        fill: black; /* Circle is black in light mode */
      }
    }

    /* Styling for dark mode */
    @media (prefers-color-scheme: dark) {
      .bg {
        fill: black; /* Background is black in dark mode */
      }
      .circle {
        fill: white; /* Circle is white in dark mode */
      }
    }
  </style>

  <rect class="bg" x="0" y="0" width="100" height="100" />
  <circle class="circle" cx="50" cy="50" r="30" />
</svg>


```

**Reply by treeandcoffee** [reply]
Yes, that worked. Many thanks.

**Comment by bobmitro** [comment]
ref: https://getpublii.com/docs/prepare-svg-for-light-dark-mode.html
